src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private Node newNode;
+    private Node currentNode;
```

Comment:
those should be local variables in methods - not fields

Reply:
and when I will define variables correctly...?